modules/navier_stokes/test/tests/auxkernels/reynolds-number-functor-aux/fv.i

```diff
@@ -0,0 +1,170 @@
+mu=1
+rho=1
+
+[GlobalParams]
+  velocity_interp_method = 'rc'
+  advected_interp_method = 'average'
+  rhie_chow_user_object = 'rc'
+[]
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 10
+    ny = 10
+  []
+[]
+
+[Variables]
+  [u]
+    type = INSFVVelocityVariable
+  []
+  [v]
+    type = INSFVVelocityVariable
+  []
+  [pressure]
+    type = INSFVPressureVariable
+  []
+  [lambda]
+    family = SCALAR
+    order = FIRST
+  []
+[]
+
+[AuxVariables]
+  [Reynolds]
+    type = MooseVariableFVReal
+  []
+[]
+
+[AuxKernels]
+  [Reynolds]
+    type = ReynoldsNumberFunctorAux
+    variable = Reynolds
+    speed = speed
+    rho = ${rho}
+    mu = ${mu}
+  []
+[]
+
+[UserObjects]
+  [rc]
+    type = INSFVRhieChowInterpolator
+    u = u
+    v = v
+    pressure = pressure
+  []
+[]
+
+[FVKernels]
+  [mass]
+    type = INSFVMassAdvection
+    variable = pressure
+    rho = ${rho}
+  []
+  [mean_zero_pressure]
+    type = FVScalarLagrangeMultiplier
+    variable = pressure
+    lambda = lambda
+  []
+
+  [u_advection]
+    type = INSFVMomentumAdvection
+    variable = u
+    rho = ${rho}
+    momentum_component = 'x'
+  []
+
+  [u_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = u
+    mu = 'mu'
+    momentum_component = 'x'
+  []
+
+  [u_pressure]
+    type = INSFVMomentumPressure
+    variable = u
+    momentum_component = 'x'
+    pressure = pressure
+  []
+
+  [v_advection]
+    type = INSFVMomentumAdvection
+    variable = v
+    rho = ${rho}
+    momentum_component = 'y'
+  []
+
+  [v_viscosity]
+    type = INSFVMomentumDiffusion
+    variable = v
+    mu = 'mu'
+    momentum_component = 'y'
+  []
+
+  [v_pressure]
+    type = INSFVMomentumPressure
+    variable = v
+    momentum_component = 'y'
+    pressure = pressure
+  []
+[]
+
+[FVBCs]
+  [top_x]
+    type = INSFVNoSlipWallBC
+    variable = u
+    boundary = 'top'
+    function = 1
+  []
+
+  [no_slip_x]
+    type = INSFVNoSlipWallBC
+    variable = u
+    boundary = 'left right bottom'
+    function = 0
+  []
+
+  [no_slip_y]
+    type = INSFVNoSlipWallBC
+    variable = v
+    boundary = 'left right top bottom'
+    function = 0
+  []
+[]
+
+[Materials]
+  [mu]
+    type = ADGenericFunctorMaterial
+    prop_names = 'mu'
+    prop_values = '${mu}'
+  []
+  [speed]
+    type = ADVectorMagnitudeFunctorMaterial
+    x_functor = u
+    y_functor = v
+    vector_functor_name = speed
+  []
+[]
+
+[Preconditioning]
+  [smp]
+    type = SMP
+    full = true
+  []
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = 'NEWTON'
+  petsc_options_iname = '-pc_type -ksp_gmres_restart -sub_pc_type -sub_pc_factor_shift_type'
+  petsc_options_value = 'asm      100                lu           NONZERO'
+  nl_rel_tol = 1e-12
+  resid_and_jacobian_together = true
```

Comment:
is this merged? I did not pay attention :(

Reply:
lol I don't know how that snuck in there

Reply:
```suggestion
```